.gitignore

```diff
@@ -1,2 +1,6 @@
 demo-dist
 node_modules
+*.log
```

Comment:
Sounds great!

I actually wanted to use something like [nanobench](https://github.com/mafintosh/nanobench) in a directory called `benchmark` at some point. But another name can do just fine. We can cross that bridge when we get there.

This LGTM